javascript/javascript1/week3/index.js

```diff
@@ -0,0 +1,84 @@
+// ***************************************************************************************************************************************
+//                                                      Item array removal
+// ***************************************************************************************************************************************
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+names.splice(names.indexOf(nameToRemove), 1);
+
+// Code done
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+
+
+// ***************************************************************************************************************************************
+//                                                      When will we be there??
+// ***************************************************************************************************************************************
+
+/*
+Write a function where you specify your speed in km/h and how far you have to go in km. The function has to return the time it will take to arrive at your destination.
+The time should be formatted like this: 3 hours and 34 minutes.
+
+Hint: For formatting your best friend is Google!
+
+*/
+
+
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+};
+
+function whenWillWeBeThere() {
```

Comment:
The function names should be a little more descriptive and precise. Try to start with a verb instead and think about what the function is doing.

Reply:
Thanks for the comment :) will more careful when it comes to naming the function from now on